generators/generator-base.js

```diff
@@ -2453,6 +2456,7 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
         dest.reactive = config.reactive;
         dest.jhiPrefix = config.jhiPrefix;
         dest.skipFakeData = config.skipFakeData;
+        dest.withGeneratedFlag = config.withGeneratedFlag;
```

Comment:
Not needed, we use this to expose configs to templates and make easier to use.
In this case we only use it at the constructor.